routers/web/web.go

```diff
@@ -234,6 +234,7 @@ func RegisterRoutes(m *web.Route) {
 	// for health check
 	m.Get("/", Home)
 	m.Get("/.well-known/openid-configuration", user.OIDCWellKnown)
+	m.Get("/.well-known/nodeinfo", NodeInfoLinks)
```

Comment:
This maybe optional because some instance don't want to be discovered.

Reply:
@lunny updated with option (and docs)